### ISSUE TITLE
Error With Network Interfaces

### ISSUE THREAD

**Issue by ctm-tech** [Azure/migAz#184]
Seem to have a issue with ASM > ARM on network interfaces wehn select Existing VNets(s) in North Europe I get the following unhandled exception:

```
Object reference not set to an instance of an object.
   at MigAz.Azure.UserControls.NetworkSelectionControl.<rbExistingARMVNet_CheckedChanged>d__19.MoveNext() in C:\Users\ruanders\Source\Repos\migAz\MigAz.Azure\UserControls\NetworkSelectionControl.cs:line 254
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.AsyncMethodBuilderCore.<>c.<ThrowAsync>b__6_0(Object state)
```

Seems to point towards someones repo.
You can still go ahead and select the Vnet you want but the migration failed and I think it all relates to this error.

**Comment by ctm-tech** [comment]
Hi,
When will you be uploading the fix for this? Said you resolved the issue in your closing statement on #176 but not seeing any new commits :)
Thanks for your help.

**Comment by rsanderson2350** [comment]
Uploading the 2.4.2.5 release now.
